integrationTests/cypress/page-objects/activities-state-staff-expenses-page.js

```diff
@@ -26,30 +27,22 @@ class ActivitiesStateStaffExpensesPage {
   };
 
   // Open the indexth staff edit page, fill info, then click done.
-  fillStaff = (staffIndex, title, desc, costs, ftes) => {
+  fillStaff = (years, staffIndex, title, description, costs, ftes) => {
```

Comment:
we should probably change this so it takes an object of these values, so it's less confusing when we call it

Reply:
good call